pagopa/pn-frontend
# PN-10957-fe-api-lista-gruppi-pg

## Short Description
Migrate the `getGroups()` API to the BFF's API.

## List of Changes Proposed in This Pull Request
- Generated models from the BFF OpenAPI.
- Updated `getGroups()` with the BFF's API.
- Updated related tests.

## How to Test
Start PG and verify that everything works as intended by going to the delegates section and checking if the API is called correctly.
